src/database/models/rating.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-vars */
+
+export default (sequelize, DataTypes) => {
+  const Rating = sequelize.define('Rating', {
+    user: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      validate: {
+        isNumeric: true,
+      },
+    },
+    quality: {
```

Comment:
since quality, quantity etc ratings, are between -2 to 2 you can add also that validation on the database